WalletWasabi.Backend/Controllers/BatchController.cs

```diff
@@ -66,7 +55,7 @@ public BatchController(BlockchainController blockchainController, ChaumianCoinJo
 			return BadRequest("Not supported index type.");
 		}
 
-		(Height bestHeight, IEnumerable<FilterModel> filters) = indexer.GetFilterLinesExcluding(knownHash, maxNumberOfFilters, out bool found);
+		(Height bestHeight, IEnumerable<FilterModel> filters) = indexer.GetFilterLinesExcluding(knownHash, 1_000, out bool found);
```

Comment:
> However, client's behavior actually differ for mainnet (1000) and testnet (10_000). So I would expect that here we do the same.

Oh! very good point. 

> As a consequence, we should remove from WasabiSynchronizer's ctor its maxFiltersToSync parameter, or not?

We should, yes. But the this change is okay anyway because the value is always 1_000 (or 10_000 testnet) and the extra parameters in the url are ignored.